cg/services/analysis_starter/tracker/tracker.py

```diff
@@ -0,0 +1,89 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from cg.apps.environ import environ_email
+from cg.apps.tb import TrailblazerAPI
+from cg.constants.constants import CustomerId
+from cg.constants.priority import TrailblazerPriority
+from cg.constants.sequencing import SeqLibraryPrepCategory
+from cg.constants.tb import AnalysisType
+from cg.exc import CgError
+from cg.meta.workflow.utils.utils import MAP_TO_TRAILBLAZER_PRIORITY
+from cg.models.cg_config import CommonAppConfig
+from cg.store.models import Case, Sample
+from cg.store.store import Store
+
+
+class Tracker(ABC):
```

Comment:
Do you think this would be used in another context than TB? otherwise adding in the name would bring clarity
```suggestion
class TrailblazerTracker(ABC):
```

Reply:
Let's keep Tracker for now